nixos/modules/services/networking/zeronsd.nix

```diff
@@ -0,0 +1,103 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
```

Comment:
we're trying to avoid this for new code, see https://nix.dev/recipes/best-practices#with-scopes

Reply:
I removed it now and just added the explicit path to `lib.` in the specific calls.